src/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import rootReducer from './reducers';
+
```

Comment:
Let's bring up our `BooksArray` in here.
![image](https://user-images.githubusercontent.com/31897434/86712209-ac405e80-c014-11ea-9d25-306d435636cb.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/